sorc/build_all.sh

```diff
@@ -35,6 +35,8 @@ Usage: ${BASH_SOURCE[0]} [-a UFS_app][-c build_config][-d][-h][-j n][-v][-w]
     Execute all build scripts with -v option to turn on verbose where supported
   -w:
     Use structured wave grid
+  -f:
```

Comment:
This should be moved up to be in alphabetical order as well.

Reply:
Done. Thanks!